src/drive/mobile/containers/RatingModal.jsx

```diff
@@ -166,48 +100,43 @@ const promptRating = ({
   softwareName,
   softwareID
 }) => {
-  return new Promise((resolve, reject) => {
-    try {
-      AppRate.preferences = {
-        displayAppName: softwareName,
-        inAppReview: true,
-        storeAppURL: {
-          ios: softwareID,
-          android: `market://details?id=${softwareID}`
-        },
-        customLocale: {
-          title,
-          message,
-          cancelButtonLabel: no,
-          laterButtonLabel: later,
-          rateButtonLabel: yes,
-          yesButtonLabel: yes,
-          noButtonLabel: no
-        },
-        callbacks: {
-          onButtonClicked: resolve
-        }
+  if (window.AppRate) {
+    window.AppRate.preferences = {
+      displayAppName: softwareName,
+      inAppReview: true,
+      storeAppURL: {
+        ios: softwareID,
+        android: `market://details?id=${softwareID}`
+      },
+      customLocale: {
+        title,
+        message,
+        cancelButtonLabel: no,
+        laterButtonLabel: later,
+        rateButtonLabel: yes,
+        yesButtonLabel: yes,
+        noButtonLabel: no
+      },
+      callbacks: {
+        onButtonClicked: Promise.resolve
       }
-
-      AppRate.promptForRating()
-    } catch (e) {
-      reject(e)
     }
-  })
+
+    window.AppRate.promptForRating()
+  } else {
+    Promise.resolve()
+  }
```

Comment:
I haven't tested the code yet, but pretty sure this change is breaking things, because `promptRating` doesn't return anything anymore, [but we want it to return](https://github.com/enguerran/cozy-drive/blob/2b00966701a7e8393d2d63054c2852035e58a436/src/drive/mobile/containers/RatingModal.jsx#L30) the result of `onButtonClicked`

Reply:
Obviously I failed my rollback ;) Well catched.